### ISSUE TITLE
Dynamic Deactivated Query

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#4109]
Dynamic Deactivated Query in the Slider and MenuSlider, like we do for name and bipolar (on the slider).

Once this is done the path to #2089 will be clear. In fact, perhaps do #2089 along with this one (but perhaps do it as two steps hence this separate issue).
